docs/docs/installation/pypi.mdx

```diff
@@ -22,6 +22,13 @@ level dependencies.
 
 **Debian and Ubuntu**
 
+Ubuntu **24.04** uses python 3.12 per default, which currently is not supported by superset. You need to add a second python installation of 3.11 and install the required additional dependencies.
+```bash
+sudo add-apt-repository ppa:deadsnakes/ppa
```

Comment:
I was unfamiliar with this repo so looked it up and found it to be credible: https://askubuntu.com/questions/1398568/installing-python-who-is-deadsnakes-and-why-should-i-trust-them